.github/CODEOWNERS

```diff
@@ -0,0 +1,28 @@
+# This file should stay in somewhat sync with the MAINTAINERS file, since there
+# is some kind of relation between the two. Preferably use GitHub alias here
+# instead of email addresses.
+
+# Everything not uniquely specified should land at the core team
+* @OP-TEE/linaro
+
+# Platform specific code
+core/arch/arm/plat-d02/ @OP-TEE/plat-d02
```

Comment:
Start all root files with `/` (according to the [example](https://help.github.com/articles/about-codeowners/#codeowners-file-example)).